src/server.h

```diff
@@ -3010,6 +3012,9 @@ void trimStringObjectIfNeeded(robj *o, int trim_small_values);
 static inline int canUseSharedObject(void) {
     return server.maxmemory == 0 || !(server.maxmemory_policy & MAXMEMORY_FLAG_NO_SHARED_INTEGERS);
 }
+static inline void calculateMaxAvailableMemory(void) {
```

Comment:
Since we are not returning any value from this function but updating a state in `server`, I feel that `update would be a better verb to indicate that this is more of "method" than "function".

```suggestion
static inline void updateMaxAvailableMemory(void) {
```

Reply:
Address